resources/lib/playback.py

```diff
@@ -564,8 +541,8 @@ def play_xml(playqueue, xml, offset=None, start_plex_id=None):
     playqueue.selectedItemID
     """
     offset = int(offset) if offset else None
```

Comment:
Convert milliseconds to seconds
```suggestion
    offset = int(offset)/1000 if offset else None
```